src/main/webapp/js/external/selfie-segmentation/selfie_segmentation.js.1

```diff
@@ -0,0 +1,75 @@
+(function(){/*
```

Comment:
Is this file necessary?

Reply:
There is a mistake, when I downloading files. Let me remove it.